nav2_behavior_tree/test/plugins/decorator/test_goal_updated_controller.cpp

```diff
@@ -36,8 +36,6 @@ class GoalUpdatedControllerTestFixture : public nav2_behavior_tree::BehaviorTree
     poses1.push_back(goal1);
     config_->blackboard->set("goal", goal1);
     config_->blackboard->set("goals", poses1);
-    config_->input_ports["goals"] = "";
```

Comment:
@SteveMacenski these were causing the test to fail, I see they were added [there](https://github.com/ros-navigation/navigation2/pull/4298/files#diff-b9f432eb1b03868819ba84ee7c21c4b98a8cc0d4a589587c46a0e0d1120d7f05R39-R40) by you I believe, do you remember what were you solving with these lines?

Reply:
There were issues in the 24.04 migration about ports not being set, so I set them all to something in the tests. It doesn't need to be `""`, but needs to be initialized as something to stop BT.CPP update from throwing an uninitialized port error due to how the tests are structured